lib/stdlib/src/gen_server.erl

```diff
@@ -692,6 +672,62 @@ See [`erlang:start_timer/4`](`erlang:start_timer/4`) for details.
 """.
 -type timeout_option() :: {abs, Abs :: boolean()}.
 
+-doc """
+Callback and server loop action.
+
+Returned by the callbacks [`Module:init/1`](`c:init/1`),
+[`Module:handle_call/3`](`c:handle_call/3`),
+[`Module:handle_cast/2`](`c:handle_cast/2`),
+and [`Module:handle_continue/2`](`c:handle_continue/2`).
+It is also the last argument to [`enter_loop/4,5`](`enter_loop/4`).
+
+`t:action/0` is one of:
+
+- **`Time :: `[`timeout()`](`t:timeout/0`)** -
```

Comment:
In the `action()` type below, this is named `Timeout`. For consistency, I suggest using either `Timeout` here, or `Time` in the type.

Reply:
I'll go for `Time` in the `action()` since that is used for `{timeout|hibernate,...}` consistently.